TLPKTDROP: explain magic number by variable. (#88)
Haivision/srt-rfc: draft-sharabayko-mops-srt.md

that have not been delivered in time. The timeout of dropping a packet is based on the 
TSBPD mechanism ({{tsbpd}}).

When TLPKTDROP is enabled, a packet timestamp is older than TLPKTDROP_THRESHOLD,
the packet is considered "Too-Late" to be delivered. Thus, the sender might need to drop
the packet.

TLPKTDROP_THRESHOLD: a marginal value that is considered that a packet is delivered in time. 
: The value is related to SRT Latency({{srt-latency}}). When considering RTT, it recommends

>   1.25 times of SRT Latency.

Reviewer: Maybe add some note that the value should be that high to allow the receiver to drop the packet first, so that sender drops packets only when serious congestion happens and it does not get a response from the peer.

Author: That's a very useful note.

that have not been delivered in time. The timeout of dropping a packet is based on the 
TSBPD mechanism ({{tsbpd}}).

> When TLPKTDROP is enabled, a packet timestamp is older than TLPKTDROP_THRESHOLD,

Reviewer: ```suggestion
With TLPKTDROP is enabled, when a packet timestamp is older than TLPKTDROP_THRESHOLD,
```

Reviewer 2: You would either say "When TLPKTDROP is enabled ..." or "With TLPKTDROP enabled ..."

Both forms are correct.

the packet is considered "Too-Late" to be delivered. Thus, the sender might need to drop
the packet.

> TLPKTDROP_THRESHOLD: a marginal value that is considered that a packet is delivered in time. 
> : The value is related to SRT Latency ({{srt-latency}}). For an efficient packet drop mechanism,
>   it is recommended higher value than the latency to allow the receiver to drop the packet first
>   so that the sender drops the packet only when no proper response is received from the peer
>   by serious congestion. When considering RTT, it recommends the threshold value has 1.25 times
>   of SRT Latency.
> 
> However, the sender keeps packets for at least 1 second in case the
> SRT latency is not enough for a large RTT (that is, if TLPKTDROP_THRESHOLD is less 

Reviewer 2: ```suggestion
TLPKTDROP_THRESHOLD: a marginal value that defines a limit for when a packet is considered to be delivered in time. 
: The value is related to SRT Latency ({{srt-latency}}). For an efficient packet drop mechanism,
  it is recommended that a higher value than the latency be used. This will allow the receiver to drop the packet first
  so that the sender drops the packet only when no proper response is received from the peer (due to
  severe congestion, for example). When considering RTT, it is recommended that the threshold value be 1.25 times
  the SRT Latency value.

Note that the sender keeps packets for at least 1 second in case the
SRT latency is not enough for a large RTT (that is, if TLPKTDROP_THRESHOLD is less 
```

for the player. Setting higher SRT latency will help much more in the
case when TLPKTDROP causes packet drops too often.

When a receiver encounters the situation where the next packet to be played was not
successfully received from the sender, the receiver will "skip" the packet

> and send a fake ACK. To the sender, this fake ACK is a real ACK({{packet-acks}}), 

Reviewer: I am not 100% sure, but likely space before the brace would make sense, as the reference renders into "Secition #.#".
Please check. One more such case is above.

```suggestion
and send a fake ACK. To the sender, this fake ACK is a real ACK ({{packet-acks}}), 
```
